docs/src/main/paradox/kubernetes-deployment/deploying.md

```diff
@@ -0,0 +1,124 @@
+# Deploying
+
+Having configured a DeploymentSpec, Role, and RoleBinding they can be created with:
+
+```
+kubectl apply -f kubernetes/akka-cluster.yaml
+```
+
+Where akka `kubernetes/akka-cluster.yaml` is location of the Kubernetes resources files in the samples.
+
+@@@note
+If you haven't been creating the files as you go for the guide, but rather are relying on the existing 
+files distributed with the sample app, make sure you have performed the following easy to miss steps:
+
+* The $spec.path$ `RoleBinding` spec @ref[needs to have the namespace updated](forming-a-cluster.md#role-based-access-control) for the user 
+  name if you are not using the `appka-1` namespace.
+@@@
+
+Immediately after running this, you should see the three pods when you run `kubectl get pods`:
+
+@@@vars
+```
+akka-sample-cluster-kubernetes-756894d68d-9sltd         0/1       Running   0          9s
+akka-sample-cluster-kubernetes-756894d68d-bccdv         0/1       Running   0          9s
+akka-sample-cluster-kubernetes-756894d68d-d8h5j         0/1       Running   0          9s
+```
+@@@
+
```

Comment:
something about trying the 8080 hello world, as we have in the readme https://github.com/akka/akka-sample-cluster-kubernetes-scala/blob/master/README.md

Reply:
and maybe also the Cluster HTTP Management, as in readme?

Reply:
I'll add that